### ISSUE TITLE
is:issue is:open You need add the validation for the fields, i can create products without name with a space and the price can are a negative number example -800

### ISSUE THREAD

**Issue by CHINOBv** [alpdias/controle-estoque-js#1]
is:issue is:open You need add the validation for the fields, i can create products without name with a space and the price can are a negative number example -800

**Comment by alpdias** [comment]
Yes, some are still missing as validations, but thanks for the tips, as soon as I mess with these codes again I will observe these parts, thanks for the collaboration!
